File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
 
 {% endtabs %} 
 
+### Link Paragraph and Character Style
+
+You can link paragraph and character style using the `LinkedStyleName` property.
```

Comment:
You can link character styles with  paragraph and vice versa in a Word document using `LinkedStyleName` property.

Reply:
changed